assets/src/components/closeButton.tsx

```diff
@@ -42,4 +42,22 @@ const CloseButton: React.FC<Props> = ({ onClick, closeButtonType }) => {
   )
 }
 
+export const CloseButton2 = ({
+  onClick,
+  closeButtonType,
+  ...rest
+}: Props & React.HTMLAttributes<HTMLElement>) => (
+  <button
+    title="Close"
+    onClick={onClick}
+    className={[
+      "m-close-button",
+      ...buttonTypeClassNames[closeButtonType],
+    ].join(" ")}
```

Comment:
**suggestion:** I think before you used the `className` function here, it might have gotten lost in the rebase?

Reply:
I was getting really annoyed with the default name of the `className` function colliding with the prop/function parameters heh, so I dropped it for the same syntax as the other.
I'm fine to pull it back in under an alias.

Reply:
Ah, makes sense. As long as it was intentional I'm fine keeping it that way. Maybe we could just find a better name for the `className` function in the future.